### DISCUSSION TITLE
About `cad`

### DISCUSSION THREAD

**Discussion by tristan-f-r** [mpromonet/webrtc-streamer#548] [Q&A] [answered]
I noticed [this interesting folder](https://github.com/mpromonet/webrtc-streamer/tree/master/cad) in the repository, and I was wondering: what is it for? There's no README attached and the model is too generic for me to guess.

**Comment by mpromonet** [comment, marked as answer]
Hi,
The cad file is my design to use https://www.thingiverse.com/thing:1998968 with a nano pi air and a camera 500B.
Best Regards,
Michel.
